inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/schema/SchemaChangeHelper.java

```diff
@@ -415,12 +417,22 @@ private String authHeader() {
     private boolean executeStatement(String database, String stmt) throws IOException {
         Map<String, String> param = new HashMap<>();
         param.put("stmt", stmt);
-        String requestUrl = String.format(SCHEMA_CHANGE_API, options.getFenodes(), database);
-        HttpPost httpPost = new HttpPost(requestUrl);
-        httpPost.setHeader(HttpHeaders.AUTHORIZATION, authHeader());
-        httpPost.setHeader(HttpHeaders.CONTENT_TYPE, CONTENT_TYPE_JSON);
-        httpPost.setEntity(new StringEntity(dynamicSchemaFormat.objectMapper.writeValueAsString(param)));
-        return sendRequest(httpPost);
+        List<String> fenodes = Arrays.asList(options.getFenodes().split(","));
```

Comment:
Will this `options.getFenodes()` be null?

Reply:
it will not, there is a precondition check during initialization:   this.fenodes = Preconditions.checkNotNull(fenodes, "fenodes  is empty");

Reply:
Okay~